log/common.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import (
+	"io"
+	"os"
+)
+
+// For debugging purposes we can change the output-writer
+var stdOut io.Writer
+var stdErr io.Writer
+
+func init() {
+	stdOut = os.Stdout
+	stdErr = os.Stderr
+}
+
+const (
+	lvlWarning = iota - 20
+	lvlError
+	lvlFatal
+	lvlPanic
+	lvlInfo
+	lvlPrint
+)
+
+// Format indicates how the prints will be shown
+var Format = FormatLvl
```

Comment:
You miss the point why I wanted to have `dbg` separated from `ui` - because it's two different use-cases: 
- `dbg` for helping in debugging
- `ui` for always interaction with the user

Now we have functions like `Print`, `Info`, ... which are used in both, what to do?

Another approach would be to have only `FormatPython` and `FormatNone`, and set `FormatLvl` by default if the `debugVisible` is > 0. What do you think?